test/suites/edgexfoundry/main_test.go

```diff
@@ -0,0 +1,42 @@
+package test
+
+import (
+	"edgex-snap-testing/test/utils"
+	"log"
+	"os"
+	"testing"
+)
+
+const platformSnap = "edgexfoundry"
+const deviceVirtualApp = "device-virtual"
+const snapAppName = platformSnap + "." + deviceVirtualApp
+const deviceVirtualDefaultServicePort = "59900"
```

Comment:
Since the numbers are increasing, you may want to group these for better go formatting.
```
const (
   x =     1
   xxx = 2
)
```